### PR TITLE
Pioneer 13 mooring deployment csvs

### PR DESCRIPTION
**Pull request by rtravis-whoi**
Included in this Pull Request are all the Pioneer 13 mooring deployment csvs for initial review.  These will be updated with deployment date/time, depth, lat/lon and any other changes after the deployments.  **Glider/AUV deploy csvs will be completed by Collin.

- ISSM - 3 engineering components are commented out because the S/Ns listed on the mooring config sheet in Vault are the same ones listed on the currently deployed ISSM mooring.  Have asked Electrical team to verify correct S/Ns.

- Associated Engineering_bulk_load updates/additions are on separate PR.

### CODE REVIEW

**Comment by snwhite** [comment]
I managed to get the WFP QCTs off of Alfresco and confirmed all of the WFP sensors S/Ns.

File: deployment/CP01CNPM_Deploy.csv [hunk 1]
```diff
@@ -17,3 +17,12 @@ AR-31,,AR-34,,CP01CNPM-WFP01-02-DOFSTK000,2,1,2018-11-02T14:40:00,2019-04-04T12:
...
+AR-39,,,,CP01CNPM-RII01-02-ADCPTG010,3,1,2019-09-25T00:00:00,,CGMCP-01CNPM-00003,,CGINS-ADCPTG-24544,,,,,,
+AR-39,,,,CP01CNPM-SBS01-00-STCENG000,3,1,2019-09-25T00:00:00,,CGMCP-01CNPM-00003,,CGCON-BSTC01-00013,,,,,,
+AR-39,,,,CP01CNPM-SBS01-01-MOPAK0000,3,1,2019-09-25T00:00:00,,CGMCP-01CNPM-00003,,CGINS-MOPAK0-11360,,,,,,
+AR-39,,,,CP01CNPM-WFP01-00-WFPENG000,3,1,2019-09-25T00:00:00,,CGMCP-01CNPM-00003,CGVEH-CWFPCP-30002,CGCON-CWFPCP-30002,,,,,,
+AR-39,,,,CP01CNPM-WFP01-01-VEL3DK000,3,1,2019-09-25T00:00:00,,CGMCP-01CNPM-00003,CGVEH-CWFPCP-30002,CGINS-VEL3DK-00470,,,,,,
+AR-39,,,,CP01CNPM-WFP01-02-DOFSTK000,3,1,2019-09-25T00:00:00,,CGMCP-01CNPM-00003,CGVEH-CWFPCP-30002,CGINS-DOFSTK-03523,,,,,,
+AR-39,,,,CP01CNPM-WFP01-03-CTDPFK000,3,1,2019-09-25T00:00:00,,CGMCP-01CNPM-00003,CGVEH-CWFPCP-30002,CGINS-CTDPFK-00185,,,,,,
+AR-39,,,,CP01CNPM-WFP01-04-FLORTK000,3,1,2019-09-25T00:00:00,,CGMCP-01CNPM-00003,CGVEH-CWFPCP-30002,CGINS-FLORTK-01538,,,,,,
+AR-39,,,,CP01CNPM-WFP01-05-PARADK000,3,1,2019-09-25T00:00:00,,CGMCP-01CNPM-00003,CGVEH-CWFPCP-30002,CGINS-PARADK-20542,,,,,,
```
Comment (cdobs) [inline deployment/CP01CNPM_Deploy.csv]: Looks good for pre-deployment

File: deployment/CP01CNSM_Deploy.csv [hunk 2]
```diff
@@ -373,3 +373,37 @@ AR-34,,,,CP01CNSM-SBD12-04-PCO2AA000,11,1,2019-04-06T14:35:00,,CGMCP-01CNSM-0001
...
+AR-39,,,,CP01CNSM-SBD11-00-DCLENG000,12,1,2019-09-25T00:00:00,,CGMCP-01CNSM-00012,,CGCON-BDCL11-20001,,,,,,
+AR-39,,,,CP01CNSM-SBD11-01-MOPAK0000,12,1,2019-09-25T00:00:00,,CGMCP-01CNSM-00012,,CGINS-MOPAK0-11843,,,,,,
+AR-39,,,,CP01CNSM-SBD11-02-HYDGN0000,12,1,2019-09-25T00:00:00,,CGMCP-01CNSM-00012,,CGINS-HYDGN0-42330,,,,,,
+AR-39,,,,CP01CNSM-SBD11-06-METBKA000,12,1,2019-09-25T00:00:00,,CGMCP-01CNSM-00012,,CGINS-METLGR-00010,,,,,,
+AR-39,,,,CP01CNSM-SBD12-00-DCLENG000,12,1,2019-09-25T00:00:00,,CGMCP-01CNSM-00012,,CGCON-BDCL12-20001,,,,,,
+AR-39,,,,CP01CNSM-SBD12-03-HYDGN0000,12,1,2019-09-25T00:00:00,,CGMCP-01CNSM-00012,,CGINS-HYDGN0-Z5971,,,,,,
+AR-39,,,,CP01CNSM-SBD12-04-PCO2AA000,12,1,2019-09-25T00:00:00,,CGMCP-01CNSM-00012,,CGINS-PCO2AA-34222,,,,,,
+AR-39,,,,CP01CNSM-SBD12-05-WAVSSA000,12,1,2019-09-25T00:00:00,,CGMCP-01CNSM-00012,,CGINS-WAVSSA-05791,,,,,,
+AR-39,,,,CP01CNSM-SBD12-06-METBKA000,12,1,2019-09-25T00:00:00,,CGMCP-01CNSM-00012,,CGINS-METLGR-00017,,,,,,
+AR-39,,,,CP01CNSM-SBD12-08-FDCHPA000,12,1,2019-09-25T00:00:00,,CGMCP-01CNSM-00012,,CGINS-FDCHPA-44902,,,,,,
```
Comment (cdobs) [inline deployment/CP01CNSM_Deploy.csv]: Looks good for pre-deployment

File: deployment/CP02PMCI_Deploy.csv [hunk 3]
```diff
@@ -107,3 +107,12 @@ AR-34,,,,CP02PMCI-WFP01-02-DOFSTK000,12,1,2019-04-17T02:25:00,,CGMCP-02PMCI-0001
...
+AR-39,,,,CP02PMCI-RII01-02-ADCPTG010,13,1,2019-09-25T00:00:00,,CGMCP-02PMCI-00013,,CGINS-ADCPTG-19151,,,,,,
+AR-39,,,,CP02PMCI-SBS01-00-STCENG000,13,1,2019-09-25T00:00:00,,CGMCP-02PMCI-00013,,CGCON-BSTC01-00017,,,,,,
+AR-39,,,,CP02PMCI-SBS01-01-MOPAK0000,13,1,2019-09-25T00:00:00,,CGMCP-02PMCI-00013,,CGINS-MOPAK0-58478,,,,,,
+AR-39,,,,CP02PMCI-WFP01-00-WFPENG000,13,1,2019-09-25T00:00:00,,CGMCP-02PMCI-00013,CGVEH-CWFPCP-99101,CGCON-CWFPCP-99101,,,,,,
+AR-39,,,,CP02PMCI-WFP01-01-VEL3DK000,13,1,2019-09-25T00:00:00,,CGMCP-02PMCI-00013,CGVEH-CWFPCP-99101,CGINS-VEL3DK-00015,,,,,,
+AR-39,,,,CP02PMCI-WFP01-02-DOFSTK000,13,1,2019-09-25T00:00:00,,CGMCP-02PMCI-00013,CGVEH-CWFPCP-99101,CGINS-DOFSTK-02978,,,,,,
+AR-39,,,,CP02PMCI-WFP01-03-CTDPFK000,13,1,2019-09-25T00:00:00,,CGMCP-02PMCI-00013,CGVEH-CWFPCP-99101,CGINS-CTDPFK-00111,,,,,,
+AR-39,,,,CP02PMCI-WFP01-04-FLORTK000,13,1,2019-09-25T00:00:00,,CGMCP-02PMCI-00013,CGVEH-CWFPCP-99101,CGINS-FLORTK-01029,,,,,,
+AR-39,,,,CP02PMCI-WFP01-05-PARADK000,13,1,2019-09-25T00:00:00,,CGMCP-02PMCI-00013,CGVEH-CWFPCP-99101,CGINS-PARADK-20437,,,,,,
```
Comment (cdobs) [inline deployment/CP02PMCI_Deploy.csv]: Looks good for pre-deployment

File: deployment/CP02PMCO_Deploy.csv [hunk 4]
```diff
@@ -98,3 +98,12 @@ AR-34,,,,CP02PMCO-WFP01-02-DOFSTK000,12,1,2019-04-22T04:09:00,,CGMCP-02PMCO-0001
...
+AR-39,,,,CP02PMCO-RII01-02-ADCPTG010,13,1,2019-09-25T00:00:00,,CGMCP-02PMCO-00013,,CGINS-ADCPTG-20495,,,,,,
+AR-39,,,,CP02PMCO-SBS01-00-STCENG000,13,1,2019-09-25T00:00:00,,CGMCP-02PMCO-00013,,CGCON-BSTC01-00016,,,,,,
+AR-39,,,,CP02PMCO-SBS01-01-MOPAK0000,13,1,2019-09-25T00:00:00,,CGMCP-02PMCO-00013,,CGINS-MOPAK0-43263,,,,,,
+AR-39,,,,CP02PMCO-WFP01-00-WFPENG000,13,1,2019-09-25T00:00:00,,CGMCP-02PMCO-00013,CGVEH-CWFPCP-99102,CGCON-CWFPCP-99102,,,,,,
+AR-39,,,,CP02PMCO-WFP01-01-VEL3DK000,13,1,2019-09-25T00:00:00,,CGMCP-02PMCO-00013,CGVEH-CWFPCP-99102,CGINS-VEL3DK-00013,,,,,,
+AR-39,,,,CP02PMCO-WFP01-02-DOFSTK000,13,1,2019-09-25T00:00:00,,CGMCP-02PMCO-00013,CGVEH-CWFPCP-99102,CGINS-DOFSTK-02499,,,,,,
+AR-39,,,,CP02PMCO-WFP01-03-CTDPFK000,13,1,2019-09-25T00:00:00,,CGMCP-02PMCO-00013,CGVEH-CWFPCP-99102,CGINS-CTDPFK-00113,,,,,,
+AR-39,,,,CP02PMCO-WFP01-04-FLORTK000,13,1,2019-09-25T00:00:00,,CGMCP-02PMCO-00013,CGVEH-CWFPCP-99102,CGINS-FLORTK-01031,,,,,,
+AR-39,,,,CP02PMCO-WFP01-05-PARADK000,13,1,2019-09-25T00:00:00,,CGMCP-02PMCO-00013,CGVEH-CWFPCP-99102,CGINS-PARADK-20447,,,,,,
```
Comment (cdobs) [inline deployment/CP02PMCO_Deploy.csv]: Looks good for pre-deployment

File: deployment/CP02PMUI_Deploy.csv [hunk 5]
```diff
@@ -107,3 +107,12 @@ AR-34,,,,CP02PMUI-WFP01-02-DOFSTK000,13,1,2019-04-15T01:03:00,,CGMCP-02PMUI-0001
...
+AR-39,,,,CP02PMUI-RII01-02-ADCPTG010,14,1,2019-09-25T00:00:00,,CGMCP-02PMUI-00014,,CGINS-ADCPTG-20499,,,,,,
+AR-39,,,,CP02PMUI-SBS01-00-STCENG000,14,1,2019-09-25T00:00:00,,CGMCP-02PMUI-00014,,CGCON-BSTC01-00042,,,,,,
+AR-39,,,,CP02PMUI-SBS01-01-MOPAK0000,14,1,2019-09-25T00:00:00,,CGMCP-02PMUI-00014,,CGINS-MOPAK0-43252,,,,,,
+AR-39,,,,CP02PMUI-WFP01-00-WFPENG000,14,1,2019-09-25T00:00:00,,CGMCP-02PMUI-00014,CGVEH-CWFPCP-71401,CGCON-CWFPCP-71401,,,,,,
+AR-39,,,,CP02PMUI-WFP01-01-VEL3DK000,14,1,2019-09-25T00:00:00,,CGMCP-02PMUI-00014,CGVEH-CWFPCP-71401,CGINS-VEL3DK-00012,,,,,,
+AR-39,,,,CP02PMUI-WFP01-02-DOFSTK000,14,1,2019-09-25T00:00:00,,CGMCP-02PMUI-00014,CGVEH-CWFPCP-71401,CGINS-DOFSTK-00209,,,,,,
+AR-39,,,,CP02PMUI-WFP01-03-CTDPFK000,14,1,2019-09-25T00:00:00,,CGMCP-02PMUI-00014,CGVEH-CWFPCP-71401,CGINS-CTDPFK-00093,,,,,,
+AR-39,,,,CP02PMUI-WFP01-04-FLORTK000,14,1,2019-09-25T00:00:00,,CGMCP-02PMUI-00014,CGVEH-CWFPCP-71401,CGINS-FLORTK-00830,,,,,,
+AR-39,,,,CP02PMUI-WFP01-05-PARADK000,14,1,2019-09-25T00:00:00,,CGMCP-02PMUI-00014,CGVEH-CWFPCP-71401,CGINS-PARADK-20525,,,,,,
```
Comment (cdobs) [inline deployment/CP02PMUI_Deploy.csv]: Looks good for pre-deployment

File: deployment/CP02PMUO_Deploy.csv [hunk 6]
```diff
@@ -116,3 +116,12 @@ AR-34,,,,CP02PMUO-WFP01-02-DOFSTK000,13,1,2019-04-23T14:11:00,,CGMCP-02PMUO-0001
...
+AR-39,,,,CP02PMUO-RII01-02-ADCPSL010,14,1,2019-09-25T00:00:00,,CGMCP-02PMUO-00014,,CGINS-ADCPSL-22846,,,,,,
+AR-39,,,,CP02PMUO-SBS01-00-STCENG000,14,1,2019-09-25T00:00:00,,CGMCP-02PMUO-00014,,CGCON-BSTC01-00005,,,,,,
+AR-39,,,,CP02PMUO-SBS01-01-MOPAK0000,14,1,2019-09-25T00:00:00,,CGMCP-02PMUO-00014,,CGINS-MOPAK0-11358,,,,,,
+AR-39,,,,CP02PMUO-WFP01-00-WFPENG000,14,1,2019-09-25T00:00:00,,CGMCP-02PMUO-00014,CGVEH-CWFPCP-10301,CGCON-CWFPCP-10301,,,,,,
+AR-39,,,,CP02PMUO-WFP01-01-VEL3DK000,14,1,2019-09-25T00:00:00,,CGMCP-02PMUO-00014,CGVEH-CWFPCP-10301,CGINS-VEL3DK-00035,,,,,,
+AR-39,,,,CP02PMUO-WFP01-02-DOFSTK000,14,1,2019-09-25T00:00:00,,CGMCP-02PMUO-00014,CGVEH-CWFPCP-10301,CGINS-DOFSTK-02708,,,,,,
+AR-39,,,,CP02PMUO-WFP01-03-CTDPFK000,14,1,2019-09-25T00:00:00,,CGMCP-02PMUO-00014,CGVEH-CWFPCP-10301,CGINS-CTDPFK-00124,,,,,,
+AR-39,,,,CP02PMUO-WFP01-04-FLORTK000,14,1,2019-09-25T00:00:00,,CGMCP-02PMUO-00014,CGVEH-CWFPCP-10301,CGINS-FLORTK-01117,,,,,,
+AR-39,,,,CP02PMUO-WFP01-05-PARADK000,14,1,2019-09-25T00:00:00,,CGMCP-02PMUO-00014,CGVEH-CWFPCP-10301,CGINS-PARADK-20464,,,,,,
```
Comment (cdobs) [inline deployment/CP02PMUO_Deploy.csv]: Looks good for pre-deployment

File: deployment/CP03ISPM_Deploy.csv [hunk 7]
```diff
@@ -17,3 +17,12 @@ AR-31,,AR-34,,CP03ISPM-WFP01-02-DOFSTK000,2,1,2018-10-23T02:55:00,2019-04-17T11:
...
+AR-39,,,,CP03ISPM-RII01-02-ADCPTG010,3,1,2019-09-25T00:00:00,,CGMCP-03ISPM-00003,,CGINS-ADCPTG-18660,,,,,,
+AR-39,,,,CP03ISPM-SBS01-00-STCENG000,3,1,2019-09-25T00:00:00,,CGMCP-03ISPM-00003,,CGCON-BSTC01-00018,,,,,,
+AR-39,,,,CP03ISPM-SBS01-01-MOPAK0000,3,1,2019-09-25T00:00:00,,CGMCP-03ISPM-00003,,CGINS-MOPAK0-58481,,,,,,
+AR-39,,,,CP03ISPM-WFP01-00-WFPENG000,3,1,2019-09-25T00:00:00,,CGMCP-03ISPM-00003,CGVEH-CWFPCP-30001,CGCON-CWFPCP-30001,,,,,,
+AR-39,,,,CP03ISPM-WFP01-01-VEL3DK000,3,1,2019-09-25T00:00:00,,CGMCP-03ISPM-00003,CGVEH-CWFPCP-30001,CGINS-VEL3DK-00468,,,,,,
+AR-39,,,,CP03ISPM-WFP01-02-DOFSTK000,3,1,2019-09-25T00:00:00,,CGMCP-03ISPM-00003,CGVEH-CWFPCP-30001,CGINS-DOFSTK-03506,,,,,,
+AR-39,,,,CP03ISPM-WFP01-03-CTDPFK000,3,1,2019-09-25T00:00:00,,CGMCP-03ISPM-00003,CGVEH-CWFPCP-30001,CGINS-CTDPFK-00184,,,,,,
+AR-39,,,,CP03ISPM-WFP01-04-FLORTK000,3,1,2019-09-25T00:00:00,,CGMCP-03ISPM-00003,CGVEH-CWFPCP-30001,CGINS-FLORTK-01537,,,,,,
+AR-39,,,,CP03ISPM-WFP01-05-PARADK000,3,1,2019-09-25T00:00:00,,CGMCP-03ISPM-00003,CGVEH-CWFPCP-30001,CGINS-PARADK-20541,,,,,,
```
Comment (cdobs) [inline deployment/CP03ISPM_Deploy.csv]: Looks good for pre-deployment

File: deployment/CP03ISSM_Deploy.csv [hunk 8]
```diff
@@ -310,3 +310,34 @@ AR-34,,,,CP03ISSM-SBD11-06-METBKA000,10,1,2019-04-07T14:42:00,,CGMCP-03ISSM-0001
...
+AR-39,,,,CP03ISSM-RID27-04-DOSTAD000,11,1,2019-09-25T00:00:00,,CGMCP-03ISSM-00011,,CGINS-DOSTAD-00385,,,,,,
+#AR-39,,,,CP03ISSM-SBC11-00-CPMENG000,11,1,2019-09-25T00:00:00,,CGMCP-03ISSM-00011,,CGCON-BCPM01-50004,,,,,,s/n same as for currently deployed ISSM
+#AR-39,,,,CP03ISSM-SBD11-00-DCLENG000,11,1,2019-09-25T00:00:00,,CGMCP-03ISSM-00011,,CGCON-BDCL11-50004,,,,,,s/n same as for currently deployed ISSM
+AR-39,,,,CP03ISSM-SBD11-01-MOPAK0000,11,1,2019-09-25T00:00:00,,CGMCP-03ISSM-00011,,CGINS-MOPAK0-43257,,,,,,
+AR-39,,,,CP03ISSM-SBD11-02-HYDGN0000,11,1,2019-09-25T00:00:00,,CGMCP-03ISSM-00011,,CGINS-HYDGN0-42316,,,,,,
+AR-39,,,,CP03ISSM-SBD11-06-METBKA000,11,1,2019-09-25T00:00:00,,CGMCP-03ISSM-00011,,CGINS-METLGR-00012,,,,,,
+#AR-39,,,,CP03ISSM-SBD12-00-DCLENG000,11,1,2019-09-25T00:00:00,,CGMCP-03ISSM-00011,,CGCON-BDCL12-50004,,,,,,s/n same as for currently deployed ISSM
+AR-39,,,,CP03ISSM-SBD12-03-HYDGN0000,11,1,2019-09-25T00:00:00,,CGMCP-03ISSM-00011,,CGINS-HYDGN0-41917,,,,,,
+AR-39,,,,CP03ISSM-SBD12-04-PCO2AA000,11,1,2019-09-25T00:00:00,,CGMCP-03ISSM-00011,,CGINS-PCO2AA-31124,,,,,,
```
Comment (cdobs) [inline deployment/CP03ISSM_Deploy.csv]: Looks good for pre-deployment

File: deployment/CP04OSPM_Deploy.csv [hunk 9]
```diff
@@ -91,3 +91,12 @@ AR-34,,,,CP04OSPM-WFP01-02-DOFSTK000,11,1,2019-04-22T14:13:00,,CGMCP-04OSPM-0001
...
+AR-39,,,,CP04OSPM-RII01-02-ADCPSL010,12,1,2019-09-25T00:00:00,,CGMCP-04OSPM-00012,,CGINS-ADCPSL-21393,,,,,,
+AR-39,,,,CP04OSPM-SBS01-00-STCENG000,12,1,2019-09-25T00:00:00,,CGMCP-04OSPM-00012,,CGCON-BSTC01-00012,,,,,,
+AR-39,,,,CP04OSPM-SBS11-02-MOPAK0000,12,1,2019-09-25T00:00:00,,CGMCP-04OSPM-00012,,CGINS-MOPAK0-11347,,,,,,
+AR-39,,,,CP04OSPM-WFP01-00-WFPENG000,12,1,2019-09-25T00:00:00,,CGMCP-04OSPM-00012,CGVEH-CWFPCP-99105,CGCON-CWFPCP-99105,,,,,,
+AR-39,,,,CP04OSPM-WFP01-01-VEL3DK000,12,1,2019-09-25T00:00:00,,CGMCP-04OSPM-00012,CGVEH-CWFPCP-99105,CGINS-VEL3DK-00212,,,,,,
+AR-39,,,,CP04OSPM-WFP01-02-DOFSTK000,12,1,2019-09-25T00:00:00,,CGMCP-04OSPM-00012,CGVEH-CWFPCP-99105,CGINS-DOFSTK-03370,,,,,,
+AR-39,,,,CP04OSPM-WFP01-03-CTDPFK000,12,1,2019-09-25T00:00:00,,CGMCP-04OSPM-00012,CGVEH-CWFPCP-99105,CGINS-CTDPFK-00169,,,,,,
+AR-39,,,,CP04OSPM-WFP01-04-FLORTK000,12,1,2019-09-25T00:00:00,,CGMCP-04OSPM-00012,CGVEH-CWFPCP-99105,CGINS-FLORTK-01466,,,,,,
+AR-39,,,,CP04OSPM-WFP01-05-PARADK000,12,1,2019-09-25T00:00:00,,CGMCP-04OSPM-00012,CGVEH-CWFPCP-99105,CGINS-PARADK-20513,,,,,,
```
Comment (cdobs) [inline deployment/CP04OSPM_Deploy.csv]: Looks good for pre-deployment

File: deployment/CP04OSSM_Deploy.csv [hunk 10]
```diff
@@ -309,3 +309,34 @@ AR-34,,,,CP04OSSM-SBD11-06-METBKA000,10,1,2019-04-05T14:55:00,,CGMCP-04OSSM-0001
...
+AR-39,,,,CP04OSSM-RID27-03-CTDBPC000,11,1,2019-09-05T00:00:00,,CGMCP-04OSSM-00011,,CGINS-CTDBPC-50108,,,,,,
+AR-39,,,,CP04OSSM-RID27-04-DOSTAD000,11,1,2019-09-05T00:00:00,,CGMCP-04OSSM-00011,,CGINS-DOSTAD-00412,,,,,,
+AR-39,,,,CP04OSSM-SBC11-00-CPMENG000,11,1,2019-09-05T00:00:00,,CGMCP-04OSSM-00011,,CGCON-BCPM01-20003,,,,,,
+AR-39,,,,CP04OSSM-SBD11-00-DCLENG000,11,1,2019-09-05T00:00:00,,CGMCP-04OSSM-00011,,CGCON-BDCL11-20003,,,,,,
+AR-39,,,,CP04OSSM-SBD11-01-MOPAK0000,11,1,2019-09-05T00:00:00,,CGMCP-04OSSM-00011,,CGINS-MOPAK0-42298,,,,,,
+AR-39,,,,CP04OSSM-SBD11-02-HYDGN0000,11,1,2019-09-05T00:00:00,,CGMCP-04OSSM-00011,,CGINS-HYDGN0-49886,,,,,,
+AR-39,,,,CP04OSSM-SBD11-06-METBKA000,11,1,2019-09-05T00:00:00,,CGMCP-04OSSM-00011,,CGINS-METLGR-00037,,,,,,
+AR-39,,,,CP04OSSM-SBD12-00-DCLENG000,11,1,2019-09-05T00:00:00,,CGMCP-04OSSM-00011,,CGCON-BDCL12-20003,,,,,,
+AR-39,,,,CP04OSSM-SBD12-03-HYDGN0000,11,1,2019-09-05T00:00:00,,CGMCP-04OSSM-00011,,CGINS-HYDGN0-49857,,,,,,
+AR-39,,,,CP04OSSM-SBD12-04-PCO2AA000,11,1,2019-09-05T00:00:00,,CGMCP-04OSSM-00011,,CGINS-PCO2AA-35264,,,,,,
```
Comment (cdobs) [inline deployment/CP04OSSM_Deploy.csv]: Looks good for pre-deployment

**Pull request merged**